docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,37 +369,41 @@ which is a CDI bean and can be ``@Inject``ed.
 [[jdbcstore]]
 == Configure storing of Quarkus transaction logs in a database
 
-In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
-
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a Java Database Connectivity (JDBC) datasource.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store.<property>` properties, where <property> can be:
+However, in cloud-native apps, using an in-database transaction store has additional requirements.
```

Comment:
Thanks @MichalMaler for modifying the `using a database to store transaction logs` at line 378. Could you please replace `using an in-database transaction store` with `using a database to store transaction logs` here as well. Thanks

Reply:
It's done, thanks @jmfinelli .

BTW I agree the GitHub UI tends to be confusing when history gets rewritten, but generally I manage by assuming no unrelated changes get introduced between reviews, and simply going through all previous conversations on the "Conversations" tab. Having the "Files changed" tab opened in a different browser tab helps :octopus:

Reply:
Thanks @yrodiere and @MichalMaler.

BTW, I see your point and thanks for the suggestion. Personally, I find long conversations hard/tricky to follow when comments disappear from "Files changed". For this reason, I usually rewrite history when all reviewers approved my PR.